### ISSUE TITLE
 'ScriptSession' object has no attribute 'windowCoord'

### ISSUE THREAD

**Issue by supermopf** [Matrixchung/EDAutopilot#19]
When starting the robigo.py via gameui.py and pressing the HomeButton I receive this error:

```
[22:18:27][WARN] Traceback (most recent call last):
  File "C:\Users\super\Desktop\EDAutopilot\scripts\robigo.py", line 203, in run
    windowCoord = session.windowCoord
AttributeError: 'ScriptSession' object has no attribute 'windowCoord', which is trying to be set.
```

The class ScriptSession does not seem to have the property 'windowCoord'
https://github.com/Matrixchung/EDAutopilot/blob/84354850eea1682b5c697ec62a28d7f0fa7ba1c8/gameui.py#L92-L146

Have a nice day,
supermopf

**Comment by Matrixchung** [comment]
Forgot to initialize the windowCoord with (windowLeftX,windowTopY) in ScriptInputMsg.
I will fix it later tomorrow :)
Thanks for issue

**Comment by Matrixchung** [comment]
Fixed in f01744de1a8a8c46ba2f5abf18de4eb02bb19912
